_sass/_toc.scss

```
@@ -0,0 +1,12 @@
+.toc {
+  background-color: #f6f7f9;
+  padding: 8px;
+}
+
+.toc > ul,li{
```

I don't think you need to set list-style-type on the `li` element.

```suggestion
.toc > ul {
```

---

Yeah I think my selector was bonkers. 1e3c01e seemed to do the trick.